### DISCUSSION TITLE
Aiming snap "Problem"

### DISCUSSION THREAD

**Discussion by Venom-V** [Sixze/ALS-Refactored#97] [Q&A] [answered]
Hey, i just started using this ALS version on my UE5 project, and i was wondering if someone could help me improve the character behaviour when stop aiming :
https://gyazo.com/dbf4d01b3cd1670a3f86a59f472a47f7

The "problem" is the way the character snap back to face toward the input i guess, instead of smoothly lowering the gun while still looking where i was aiming (Like in base ALS). https://gyazo.com/94e55859f2e424eda44c482891cc6b22

Thanks !

**Comment by Sixze** [comment, marked as answer]
Fixed in 6e64615a19f4342a079b7afc4f54b6f7d5adaa3f.

**Reply by Venom-V** [reply]
Thanks it did patch it when being idle, but it still does it when turn/rotate in place. https://gyazo.com/5127822330e45f79d6d36b40dfa5a35a

**Reply by Sixze** [reply]
Unfortunately, I could not find a reliable solution to this issue: it is not significant and fixing it breaks other more important things, so I'll leave it as it is.

**Reply by Venom-V** [reply]
That's alright, thank you.
